microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -94,24 +203,59 @@ https://login.microsoftonline.com/5c9eedce-81bc-42f3-8823-48ba6258b391/oauth2/v2
 }
 ```
 
-For more information, see [Get a token](/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow#get-a-token) in _Microsoft identity platform and the OAuth 2.0 client credentials flow_.
+For more information, see [Get a token](/azure/active-directory/develop/v1.1-oauth2-client-creds-grant-flow#get-a-token) in _Microsoft identity platform and the OAuth 2.0 client credentials flow_.
+
+---
 
 
 <!-- ====================================================================== -->
 ## Using the API endpoints
 
-After you have an access token, you can use the Microsoft Edge Add-ons API.  This API exposes endpoints for getting a list of products, updating products, and publishing products.
+The Update REST API exposes endpoints for:
+* [Uploading a package to update an existing submission](#uploading-a-package-to-update-an-existing-submission)
+* [Checking the status of a package upload](#checking-the-status-of-a-package-upload)
+* [Publishing the submission](#publishing-the-submission)
+* [Checking the publishing status](#checking-the-publishing-status)
+
+These endpoints are introduced below.
+
+To use this Microsoft Edge Add-ons Update REST API, you must have the following, per above:
+
+##### [v1.1](#tab/v1-1)
+
+An API key and a Client ID.
+
+##### [v1](#tab/v1)
+
+An access token.
+
+---
 
-> [!NOTE]
-> There's no API for creating a new product or updating a product's metadata. For example, the description.  You must complete these tasks manually in Microsoft Partner Center.
+The API is available at the endpoint `https://api.addons.microsoftedge.microsoft.com`.
```

Comment:
is that an endpoint?
https://api.addons.microsoftedge.microsoft.com
 vs eg /v1/products/$productID/submissions/draft/package

i'm not set up - here's what i see if i go to that url (not intended usage):
![image](https://github.com/user-attachments/assets/6504de77-1aaf-43ea-aed5-1ae5b12991d8)

Reply:
result:
```
The API is available at the endpoint root `https://api.addons.microsoftedge.microsoft.com`.
```